hack/logcheck.out

```diff
@@ -1 +1,3 @@
+/pkg/reconciler/scheduling/placement/controller.go:209:3: Key positional arguments "Namespace" are expected to no special characters. Please refer to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments.
+/pkg/reconciler/scheduling/placement/controller.go:237:3: Key positional arguments "Location" are expected to no special characters. Please refer to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments.
```

Comment:
@waltforme : you may be interested in these new complaints from logcheck

Reply:
See the Kubernetes norms in https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments .  They call for the keys to be lowerCamelCase.